docs/DeveloperGuide.md

```diff
@@ -7,26 +7,42 @@ title: Developer Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Acknowledgements**
+## **Acknowledgements** (OUTDATED)
 
 * {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Setting up, getting started**
+## **Setting up, getting started** (OUTDATED)
 
 Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Design**
 
+RC4HDB has the idea of providing a flexible product which functions well, yet not too complex. Keeping this in mind, 
```

Comment:
This sentence sounds weird to me, perhaps we could change it to "**RC4HDB is designed with the intention of providing a flexible product which functions well and is at the same time, not too complex.**"